villainary-react/app/profile/page.tsx

```diff
@@ -1,5 +1,7 @@
 import { Container } from '@mui/material';
-import Profile from './Profile';
+import dynamic from "next/dynamic";
```

Comment:
what does this do?

Reply:
This is how next.js handles lazy loading of components.  It's also allowed me to explicitly label components as client side components.  Read more here: https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading

Reply:
is this different than "use client"?